documentation/modules/ref-configuring-container-images.adoc

```diff
@@ -18,22 +18,35 @@ Container image which should be used for given components can be specified using
 * `KafkaConnectS2I.spec`
 * `KafkaBridge.spec`
 
-== Configuring the `Kafka.spec.kafka.image` property
+== Configuring the `image` property for Kafka, Kafka Connect and Kafka Mirror Maker
 
```

Comment:
Please can you re-word the lead-in sentence for the bullet list.

EXISTING: Container image which should be used for given components can be specified using the `image` property in:

NEW: You can specify which container image to use for each component using the `image` property in the following resources: